src/main/java/com/microsoft/azure/datalake/store/RequestOptions.java

```diff
@@ -30,4 +30,6 @@
      * the {@link RetryPolicy} to use for the request
      */
     public RetryPolicy retryPolicy = null;
+
+    public String contentType;
```

Comment:
We should be able to do this without having to change the public interface at all. You can add an attribute to the Operation enum for content-type (and define constants C.mimeTypeNull and mimeTypeJson). This will keep the metada-data within the SDK, and require no customer input for Core. Since the mime-type is fixed (not really changeable by customer in SDK), it should not be a customer-specified parameter.